src/App.tsx

```diff
@@ -1,24 +1,46 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoContent } from './components/TodoContent';
+import { Notifications } from './components/Notifications';
+import { todosApi } from './api/todos-api';
+import { useTodoContext } from './context/todoContext/useTodoContext';
+import { useErrorContext } from './context/errorContext/useErrorContext';
 
-const USER_ID = 0;
+const USER_ID = 10875;
 
 export const App: React.FC = () => {
+  const { setTodos } = useTodoContext();
+  const { errorMessage, notifyAboutError } = useErrorContext();
+
+  useEffect(() => {
+    todosApi.getByUser(USER_ID)
+      .then(setTodos)
+      .catch(() => {
+        notifyAboutError('Something went wrong');
+      });
+  }, []);
+
   if (!USER_ID) {
```

Comment:
The condition `!USER_ID` will never be true because `USER_ID` is a constant and it's not zero. Either make `USER_ID` dynamic or remove this condition if it's not required.